designsystem/src/main/java/com/giraffe/designsystem/composable/BaseBottomSheet.kt

```diff
@@ -36,14 +39,14 @@ fun BaseBottomSheet(
         ModalBottomSheet(
             onDismissRequest = onDismiss,
             sheetState = rememberModalBottomSheetState(),
-            modifier = modifier,
+            modifier = modifier.padding(WindowInsets.navigationBars.asPaddingValues()),
```

Comment:
you can use this modifier
```suggestion
            modifier = modifier.navigationBarsPadding(),
```